app/gui2/src/components/widgets/DropdownWidget.vue

```diff
@@ -12,6 +13,14 @@ enum SortDirection {
 const props = defineProps<{ color: string; selectedValue: string | undefined; values: string[] }>()
 const emit = defineEmits<{ click: [index: number, keepOpen: boolean] }>()
 
+const tree = injectWidgetTree()
+
+let endClippingInhibition: (() => void) | undefined
+onMounted(() => {
+  endClippingInhibition = tree.inhibitClipping()
+})
+onUnmounted(() => endClippingInhibition?.())
+
```

Comment:
This should be part of WidgetSelection IMO. DropDownWidget itself may be a general-usage widget, and be used outside GraphNode (so there will be no injectWidgetTree)